package.json

```diff
@@ -58,7 +58,7 @@
     "mongodb": "^1.4.40",
     "morgan": "^1.5.1",
     "object-assign": "^2.0.0",
-    "@centerforopenscience/osf-style": "^1.3.2",
+    "@centerforopenscience/osf-style": "git://github.com/CenterForOpenScience/osf-style.git#6e1b6fb35a5dfd03530f2b8e55e0a27e01b9d080",
```

Comment:
This commit has been merged in to osf-style as of 1.4.0

Reply:
This should be updated to osf-style ^1.5.1 to include the latest padding fixes for the navbar